core/src/main/java/org/apache/iceberg/view/ViewMetadata.java

```diff
@@ -257,8 +258,8 @@ public Builder addVersion(ViewVersion version) {
       return this;
     }
 
-    private int addVersionInternal(ViewVersion version) {
-      int newVersionId = reuseOrCreateNewViewVersionId(version);
+    private int addVersionInternal(ViewVersion newVersion) {
```

Comment:
I've renamed this parameter to make the code slightly easier to understand, because we have a few cases in this method where we re-assign either the schema id or the view version id